sdk/src/main/java/com/silanis/esl/sdk/service/SigningStyleService.java

```diff
@@ -96,4 +100,69 @@ public void deleteSigningThemes() {
             throw new EslException("Could not update the signing themes to account.", e);
         }
     }
+
+    /**
+     * Save account signing logos.
+     *
+     * @param signingLogos
+     */
+    public void createSigningLogos(List<SigningLogo> signingLogos) {
+        String path = template.urlFor(UrlTemplate.ACCOUNT_SIGNING_LOGO_PATH).build();
+        String payload = JacksonUtil.serialize(signingLogos);
+        try {
+            client.post(path, payload);
+        } catch (RequestException e) {
+            throw new EslServerException("Could not save the signing logos for account.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not save the signing logos for account.", e);
+        }
+    }
+
+    /**
+     * Update account signing logos.
+     *
+     * @param updatedSigningLogos
+     */
+    public void updateSigningLogos(List<SigningLogo> updatedSigningLogos) {
+        String path = template.urlFor(UrlTemplate.ACCOUNT_SIGNING_LOGO_PATH).build();
+        String payload = JacksonUtil.serialize(updatedSigningLogos);
+        try {
+            client.post(path, payload);
+        } catch (RequestException e) {
+            throw new EslServerException("Could not save the signing logos for account.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not save the signing logos for account.", e);
+        }
+    }
+
+    /**
+     * Delete account signing logos.
+     */
+    public void deleteSigningLogos() {
+        String path = template.urlFor(UrlTemplate.ACCOUNT_SIGNING_LOGO_PATH).build();
+        String payload = JacksonUtil.serialize(new ArrayList());
+        try {
+            client.post(path, payload);
+        } catch (RequestException e) {
+            throw new EslServerException("Could not delete the signing logos for account.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not delete the signing logos for account.", e);
+        }
+    }
+
```

Comment:
All three methods above are the same method. All three use the same API path with the same POST method, so unify it in one method.
If you want to separate the three, please separate the update and delete into separate APIs in the public API and use the `PUT` and `DELETE` methods respectively.